tests/lang.sh

```diff
@@ -4,6 +4,7 @@ export TEST_VAR=foo # for eval-okay-getenv.nix
 export NIX_REMOTE=dummy://
 
 nix-instantiate --eval -E 'builtins.trace "Hello" 123' 2>&1 | grep -q Hello
+nix-instantiate --eval -E 'builtins.addErrorContext "Hello" 123' 2>&1
```

Comment:
Accidentally committed?

Reply:
not at all, the next commands again tests for errors. But such errors could be a crash, and we only care about the grep error. 
Unless the return code of pipelines handles that ?

Reply:
Oh you’re right, the tests don’t set `pipefail`. So that makes total sense